pandas/plotting/_matplotlib/core.py

```diff
@@ -784,6 +784,8 @@ def _adorn_subplots(self, fig: Figure) -> None:
             # ylabel will set it as ylabel in the plot.
             if self.ylabel is not None:
                 ax.set_ylabel(pprint_thing(self.ylabel))
+            else:
```

Comment:
This seems pretty hacky and the comment preceding it seems wrong. Do you see where this is getting set at a deeper level?

Reply:
Yes, it's defining the ylabel as the series name by default in `PiePlot` class.
Will remove that logic and let it be controlled by the ylabel parameter.